calculatorLib/src/main/java/com/example/calculatorlib/Calculator.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.calculatorlib
+
+
+class Calculator {
+    fun evaluate(str: String?): Int {
+        // 문자열 수식(str)이 null일 경우 IllegalArgumentException 발행
+        val numAndOperator = str?.split(" ") ?: throw IllegalArgumentException()
```

Comment:
https://github.com/next-step/android-calculator-mvp/pull/196#discussion_r1278302646

Exception 객체를 생성할 때 에러 메시지를 담아보라는 의미었습니다 :)
아래는 참고로 활용해주세요!
```suggestion
        // 문자열 수식(str)이 null일 경우 IllegalArgumentException 발행
        val numAndOperator = str?.split(" ") ?: throw IllegalArgumentException("입력 수식이 null이거나 빈 문자열이어서는 안됨. 입력 값 : $str")
```